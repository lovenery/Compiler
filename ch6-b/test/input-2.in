H2O + CO -> ABC2D2
