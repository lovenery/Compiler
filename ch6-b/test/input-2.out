A -1
B -1
C -1
D -2
H 2
O 2
